content/influxdb/cloud/account-management/pricing-plans.md

```diff
@@ -53,38 +54,43 @@ The Usage-based Plan offers more flexibility and ensures you only pay for what y
 
 ### Pricing vectors
 
-The following two vectors determine pricing:
+The following vectors determine pricing:
 
 - **Data out** is the total sum of the data (measured in GB) returned to the user to answer a query, also known as data transfer costs.
-- **Query count** is the total number of individual query operations. Each individual operation — including queries, tasks, alerts, notifications, and Data Explorer activity — is one billable query operation. Refreshing a dashboard with multiple cells will incur multiple query operations. Failed operations aren’t counted.
-- **Data In** is the amount of data you’re writing into InfluxDB (measured in MB).
+- **Query count** is the total number of individual query operations:
+   - Each individual operation—including queries, tasks, alerts, notifications, and Data Explorer activity—is one billable query operation.
+   - Refreshing a dashboard with multiple cells will incur multiple query operations. 
+   - Failed operations aren’t counted.
+- **Data In** is the amount of data you’re writing into InfluxDB (measured in MB/second).
 - **Storage** is the amount of data you’re storing in InfluxDB (measured in GB/hour).
 
-### Usage-Based Plan rate limits
+### Usage-Based Plan limits
 
-To protect against any intentional or unintentional harm, Usage-Based Plans include soft rate limits:
+To protect against any intentional or unintentional harm, Usage-Based Plans include soft limits.
+_To request higher rate limits, contact [InfluxData Support](mailto:support@influxdata.com)._
 
 - **Data In:** 300MB every 5 minutes
+- **Query:** 3000MB every 5 minutes
 - **Ingest batch size:** 50MB
-- **Queries:** 3000MB every 5 minutes
+- **Series cardinality:** 1,000,000 initial limit (higher limits available; [contact InfluxData Support](mailto:support@influxdata.com))
 - **Storage:** Unlimited retention
 {{% note %}}
 Increase the retention period up to 1 year by [updating a bucket’s retention period in the InfluxDB UI](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-retention-period-in-the-influxdb-ui), or [set a custom retention period](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-retention-period) using the [`influx` CLI](influxdb/cloud/reference/cli/influx/).
```

Comment:
We can increase bucket retention period to Unlimited or up to 1 year by updating the bucket's retention in the InfluxDB UI. For custom retention periods use CLI.